lib/sshkit/ssh/channel.ex

```diff
@@ -266,4 +266,8 @@ defmodule SSHKit.SSH.Channel do
   end
 
   defp ljust(_, _), do: :ok
+
+  defp ssh_connection_module(conn) do
```

Comment:
i agree in general, but in this case (i think) `ssh_connection` is good, because it normally references the erlang `:ssh_connection` module. I'm not talking about our connection-struct here.